packages/react-instantsearch-dom-maps/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-instantsearch-dom-maps",
-  "version": "6.38.1",
+  "version": "7.0.0",
```

Comment:
I wonder what concluded this has a breaking change? probably an inconspicuous commit?

Reply:
I'm thinking it counts all of the migrated commits as part of the release, including those who are flagged as breaking.